core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -688,7 +688,7 @@ object KafkaConfig {
   val FetchPurgatoryPurgeIntervalRequestsDoc = "The purge interval (in number of requests) of the fetch request purgatory"
   val ProducerPurgatoryPurgeIntervalRequestsDoc = "The purge interval (in number of requests) of the producer request purgatory"
   val DeleteRecordsPurgatoryPurgeIntervalRequestsDoc = "The purge interval (in number of requests) of the delete records request purgatory"
-  val AutoLeaderRebalanceEnableDoc = "Enables auto leader balancing. A background thread checks and triggers leader balance if required at regular intervals"
+  val AutoLeaderRebalanceEnableDoc = "Enables auto leader balancing. A background thread checks the distribution of partition leaders at regular intervals, configurable by `leader.imbalance.check.interval.seconds`. If required, it triggers a leader rebalance to the preferred leader for partitions"
```

Comment:
May be we can also include the config `leader.imbalance.per.broker.percentage` in the description.

If it exceeds the `leader.imbalance.per.broker.percentage`, it triggers a leader...

Reply:
Yes I think that makes sense, I'll look at re-writing that into the description today